test/name_service/api.clj

```diff
@@ -32,4 +35,72 @@
          "X-XSS-Protection" "1; mode=block"
          "X-Download-Options" "noopen"
          "X-Permitted-Cross-Domain-Policies" "none"
-         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
\ No newline at end of file
+         "Content-Security-Policy" "object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;"})))
+
+(deftest test-delete-keymap-wrong-params
```

Comment:
I think names of your tests are not descriptive enough. I think something like `test-delete-keymap-returns-400-when-params-are-wrong`